### ISSUE TITLE
It asks to loggin on everything I do

### ISSUE THREAD

**Issue by ghost** [TabularEditor/TabularEditor3#1408]
### Description

When I'm connected to a dataset in a workspace with premium per user, it's asking me to login after everything I do.
Saving changes, executing queries, open relationships. Basically everything I do, sometimes it even ask me multiple times in a row to login.

### Tabular Editor 3 Version

3.18.2.5824

**Comment by otykier** [comment]
@ManuelMendozaInterevoluc does this happen also if you use the "Integrated" authentication option to connect, or only with "Microsoft Entra MFA"?

<img width="417" alt="image" src="https://github.com/user-attachments/assets/c1884750-b6cf-48cc-bf52-a0ff53a43913">

**Comment by Interevoluc** [comment]
> @ManuelMendozaInterevoluc does this happen also if you use the "Integrated" authentication option to connect, or only with "Microsoft Entra MFA"?
> 
> <img alt="image" width="417" src="https://private-user-images.githubusercontent.com/8976200/393170778-c1884750-b6cf-48cc-bf52-a0ff53a43913.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQwMzI5MjEsIm5iZiI6MTczNDAzMjYyMSwicGF0aCI6Ii84OTc2MjAwLzM5MzE3MDc3OC1jMTg4NDc1MC1iNmNmLTQ4Y2MtYmY1Mi1hMGZmNTNhNDM5MTMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIxMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMTJUMTk0MzQxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MzNlOWYwZTNiMjg2ZTNkNDE1NDM5ZDhlMzI3MzJlMzc1M2ZhZjFhYTkwZWZjNWFkNmUyZjkzYTYyNDkyZTY5NiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.D3WdG3Jq9ZUNlETyV7r6HWrqXo_Q51gXqyhnWMx9Zb0">

You are right... it only happens with integrated, MFA seems to be working right.
